### PR TITLE
54954: document new "Enable SMB1 field"

### PR DESCRIPTION
**Pull request by Mrt134**
- Replace related images.
- Add note to troubleshooting SMB section.
- HTML build testing: no issues.

### CODE REVIEW

File: userguide/services.rst [hunk 1]
```diff
@@ -1456,6 +1456,11 @@ screen is really a front-end to
    | Description                 | string            | Enter an optional server description.                                                                |
    |                             |                   |                                                                                                      |
    +-----------------------------+-------------------+------------------------------------------------------------------------------------------------------+
+   | Enable SMB1 support         | checkbox          | Allow legacy SMB clients to connect to the server. **Warning:** SMB1 has been deprecated as it is    |
```
Comment (wblock) [inline userguide/services.rst]: This should be more specific: "SMB1 is not secure and has been deprecated by Microsoft."

File: userguide/services.rst [hunk 2]
```diff
@@ -1679,6 +1684,23 @@ unless there is a specific need.**
   overhead.
 
 
+Check if SMB1 support is needed by opening the :ref:`Shell` and
```
Comment (wblock) [inline userguide/services.rst]: This is a misunderstanding.  If server min protocol is set to NT1 in the file, it means the user has enabled SMB1, not that it is required.

The wording needs to be more like "Some older clients might require enabling SMB1..." and then the standard warnings and a link to the Why-Not-SMB1 article.

**Comment by Mrt134** [comment]
Checking with latest 11.2-INTERNAL .iso shows "Enable SMB1" field not present in new UI as of 1/10/19.
Waiting for PRs related to https://redmine.ixsystems.com/issues/56013 to be merged before creating angulargui branch PR

File: userguide/services.rst [hunk 3]
```diff
@@ -1679,6 +1684,12 @@ unless there is a specific need.**
   :ref:`share settings <smb_share_opts_tab>`. Many have performance
   overhead.
 
+.. needs link to smb1 knowledgebase article, when it is published. Adjust title of link, if necessary.
+
+The SMB1 protocol is deprecated and vulnerable. Some older clients still
+require it. Before enabling SMB1, see `Why You Should Not Use SMB1 <>`__
```
Comment (Mrt134) [inline userguide/services.rst]:
```suggestion
require it. Before enabling SMB1, see `Why You Should Not Use SMB1
<https://www.ixsystems.com/blog/library/do-not-use-smb1/>`__
```

**Pull request merged**